engine/pikachu/pikachu_emotions.asm

```diff
@@ -184,7 +211,35 @@ DeletedFunction_fcffb:
 REPT 5
 	nop
 ENDR
+IF DEF(_DEBUG)
+	push hl
+	ld hl, wd732
+	bit 1, [hl]
+	pop hl
+	ret z
+	push de
+	ld d, a
+	ld a, [wCurMap]
+	cp REDS_HOUSE_2F
+	ld a, d
+	pop de
+	ret nz
+	ld a, [wExpressionNumber]
+	inc a
+	cp (PikachuEmotion33_id - PikachuEmotionTable) / 2
```

Comment:
There might be a cleaner way to write this

Reply:
I'm sure the multiple overworld Pikachu-related data tables, constant lists, and macro sequences could all be cleaned up somewhat. This looks fine for now.